src/Uno.UI/UI/Xaml/Controls/ContentPresenter/ContentPresenter.wasm.cs

```diff
@@ -17,9 +17,17 @@ namespace Windows.UI.Xaml.Controls
 	/// </remarks>
 	public partial class ContentPresenter : FrameworkElement
 	{
+		private readonly BorderLayerRenderer _borderRenderer;
+
 		public ContentPresenter()
 		{
+			_borderRenderer = new BorderLayerRenderer(this);
+
 			InitializeContentPresenter();
+
+			Loaded += (s, e) => UpdateBorder();
+			Unloaded += (s, e) => _borderRenderer.Clear();
+			LayoutUpdated += (s, e) => UpdateBorder();
```

Comment:
Maybe this code could be directly in the `BorderLayerRenderer` instead of being recoded each time it is used.